bin/csv_from_oai.rb

```diff
@@ -0,0 +1,80 @@
+require 'csv'
+require 'oai'
```

Comment:
Reminder fo later; we'll need to ensure that we're loading the `./lib/oai/client_decorator.rb` as it contains the necessary overrides to account for the unique OAI implementation of the existing server.

We'll want to add:

```ruby
require_relative "../lib/oai/client_decorator"
```

Reply:
Added to code